src/test.cpp

```diff
@@ -84,8 +84,13 @@ int main(int argc, char **argv)
 
 	vfunc SDL2 = [](strs &c, strs &l)

```

Comment:
The variable could be renamed `SDL`, and there's a `"SDL2"` string in the return below that could be `#if`'d.

Reply:
Updated the PR.